### PR TITLE
disallowEmptyBlocks: allow blocks with comments

### LINKED ISSUE #1611: Add option to disallowEmptyBlocks to allow blocks with comments
https://github.com/jscs-dev/node-jscs/blob/master/lib/rules/disallow-empty-blocks.js

I'd like this to be allowed:

```
if (foo) {
  ...
} else if (bar) {
  // We shouldn't do anything here because of ...
} else {
  ...
}
```

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25336551-add-option-to-disallowemptyblocks-to-allow-blocks-with-comments?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by faceleg**
Requires rule to be configured with `allExcept: ['comments']`

Fixes #1611

### CODE REVIEW

File: test/specs/rules/disallow-empty-blocks.js [hunk 1]
```diff
@@ -125,4 +125,37 @@ describe('rules/disallow-empty-blocks', function() {
     it('should not report empty block for function expressions', function() {
         assert(checker.checkString('var a = function(){};').isEmpty());
     });
+
+    describe('options as object', function() {
+        describe('allExcept as option', function() {
```
Comment (markelog) [inline test/specs/rules/disallow-empty-blocks.js]: It seems you could unite those two describes, also, could you call it `allExcept: ["comments"]`, we do that in couple places, i guess that would be good trend to follow.

File: lib/rules/disallow-empty-blocks.js [hunk 2]
```diff
@@ -34,10 +34,21 @@ module.exports = function() {};
...
 
     configure: function(options) {
-        assert(
-            options === true,
-            this.getOptionName() + ' option requires a true value or should be removed'
-        );
+        if (typeof options !== 'object') {
+            assert(
+                options === true,
+                this.getOptionName() + ' option requires a true value or should be removed'
```
Comment (markelog) [inline lib/rules/disallow-empty-blocks.js]: Hm, i guess we should change that warning now?

**Comment by markelog** [comment]
Could you also update documentation on top of the rule file?

File: test/specs/rules/disallow-empty-blocks.js (hunk 1 above)
Comment (faceleg) [inline test/specs/rules/disallow-empty-blocks.js]: Good idea, I copied most of this from disallow-operator-before-line-break.

**Comment by faceleg** [comment]
@markelog done, let me know if anything else needs to change

**Comment by markelog** [comment]
Thank you!

**Pull request closed without merging**